### ISSUE TITLE
7 as $min-contrast-ratio is not working

### ISSUE THREAD

**Issue by stijn7621** [twbs/bootstrap#38138]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

I have for example buttons with color like #5fc8b4, #e64b78. However the text inside those buttons shows up in black instead of white which i expect. I'm not possible to set another value higher than 4.5 for $min-contrast-ratio. $min-contrast-ratio: 7 should be a valid one, according to the comment above the rule, but it isn't compiling.

![afbeelding](https://user-images.githubusercontent.com/26278060/221917505-8dcfe2dc-2d82-4881-a16d-fbd0234fa5f0.png)

### Reduced test cases

.

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Safari, Firefox, Microsoft Edge, Opera

### What version of Bootstrap are you using?

5.3.0-alpha1

**Comment by julien-deramond** [comment]
Thanks for reporting this issue @stijn7621. If I have well understood the issue, you can't compile if you set `7` to `$min-contrast-ratio` custom Sass build?

Tried it locally in the Bootstrap repo with our color and with `$min-contrast-ratio:   7 !default;` and it seems to compile well. I just got some warnings telling me that no color leading to 7:1 contrast ratio against some colors was found.

**Comment by stijn7621** [comment]
I am able to compile, however i get a lot of those 
repetitive warnings in my webpack build.

**Comment by stijn7621** [comment]
The issue still exists, however I didn't know that I had to decrease the variable instead in bs5. I changed it to 2, and everything seems to work well. This answer on stackoverflow https://stackoverflow.com/questions/68298372/customize-bootstrap-5-text-color-contrast-in-button helped me a lot.

**Comment by mdo** [comment]
Yeah, the color contrast functions are still WIP with the WCAG as far as I know. They're even working on a brand new algorithm to replace what we're using today, so hopefully that gets better soon.

If there's suggested improvements to the docs we can make, please open a new issue or PR and we'll review. Thanks!
